src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToResourceTransformers.kt

```diff
@@ -0,0 +1,99 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
```

Comment:
Just slightly confused as to what this transform is doing in comparison to the transform that you currently have in the Licence. Should they be in one file?

Reply:
think this should sit in `package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.publicApi`
Could maybe call it `LicenceDetailTransformer`